tox.ini

```diff
@@ -79,6 +79,7 @@ commands =
 changedir = {toxinidir}
 deps = -rrequirements/docs.txt
 commands =
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
     python -m sphinx.cmd.build -P -b {posargs:html} docs/source docs/build/{posargs:html}
```

Comment:
An alternative could be to put this in `requirements/docs?` What part of the docs requires a feature from the core package? And does it matter what version is installed?

Reply:
`autosummary` has to import the python modules so it can read the docstrings. The `nvtabular` modules import modules from `merlin-core`, which wasn't included in the tox environment, so it raised a "failed import" error.

Reply:
If in general the docs environment need to have all modules imported installed, then would it make sense to do a `pip install .` before the build so that anything specified in the package `install_requires` get's installed first?

Reply:
I think this makes sense. I added it in a new commit.